### PR TITLE
Added support to multi template webserver trought config file

### PR DESCRIPTION
**Pull request by LeaderGL**
## Description

Added support to multiple templates for the webserver trought the configuration file.
User can select the target template setting the 'customWebServerTemplate' field.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [N/A] **I have read CONTRIBUTING.md**
- [N/A] **I fully understand [Github Flow.](https://guides.github.com/introduction/flow/)**
- [x] **My code adheres to the [code style of this project.](https://poloniexlendingbot.readthedocs.io/en/latest/contributing.html)**
- [N/A] **I have updated the documentation in /docs if I have changed the config, arguments, logic in how the bot works, or anything that understandably needs a documentation change.**
- [x] **I have updated the config file accordingly if my change requires a new configuration setting or changes an existing one.**
- [x] **I have tested the bot with no issues for 24 continuous hours. If issues were experienced, they have been patched and tested again.**

### CODE REVIEW

**Comment by rnevet** [comment]
Hi,
So if I understand correctly this should allow you to run different HTML/JS templates for presenting the bot status?  Do you have an example other then default already?
Could the default be the /www and not /www/default? as this change breaks current config as far as I can see.

**Comment by LeaderGL** [comment]
rvenet yes this is the reason why. 
As test templates i've used the default one and the one, under pull request, of tamaskan (with some changes).
The default value can be whatever the user want, i use /www/default in order to have a template tree and you're right this broke the current config. I could use /www as default but this is a little bit dirty, any other suggestion?

**Comment by LeaderGL** [comment]
updated code in order to no break old configurations

File: www/manifest.json [hunk 1]
```diff
@@ -1,19 +1,19 @@
-{
```
Comment (rnevet) [inline www/manifest.json]: Why are there changes to this file?

Comment (LeaderGL) [inline www/manifest.json]: this is very strange, i've only "moved" back all the files from /www/default to /www without changing them. furthermore no ones of the other files in the same directory looks changed

**Comment by LeaderGL** [comment]
documentation file updated

File: www/manifest.json (hunk 1 above)
Comment (LeaderGL) [inline www/manifest.json]: also the commit log show the file as renamed without changes (https://github.com/BitBotFactory/poloniexlendingbot/pull/417/commits/ec485cc0f9ee86b4af3cf43c7bdf48db24074dc5)

File: modules/WebServer.py [hunk 2]
```diff
@@ -4,29 +4,47 @@
...
-def initialize_web_server(config):
-    import threading
-    global web_server_ip, web_server_port
-
-    if config.has_option('BOT', 'customWebServerAddress'):
-        custom_web_server_address = (config.get('BOT', 'customWebServerAddress').split(':'))
-        if len(custom_web_server_address) == 1:
-            custom_web_server_address.append("8000")
-            print "WARNING: Please specify a port for the webserver in the form IP:PORT, default port 8000 used."
+def get_web_server_setting(config, option, default):
```
Comment (rnevet) [inline modules/WebServer.py]: config.get already has a default value option this function is not required:
https://github.com/BitBotFactory/poloniexlendingbot/blob/master/modules/Configuration.py#L44

File: modules/WebServer.py [hunk 3]
```diff
@@ -4,29 +4,47 @@
...
+
+def initialize_web_server(config):
+    '''
+    Setup the web server, retrieving the configuration parameters
+    and starting the web server thread
+    '''
+    global web_server_ip, web_server_port, web_server_template
+
+    # Check for custom web server address
+    web_server_ip = get_web_server_setting(config, 'customWebServerAddress', '0.0.0.0')
```
Comment (rnevet) [inline modules/WebServer.py]: Can we find an elegant way not to break current config, or at least issue an error if web_server_ip contains the port information as it is in the current config?

File: www/manifest.json (hunk 1 above)
Comment (rnevet) [inline www/manifest.json]: maybe whitespace difference, if you compare with master do you get a difference?

Comment (LeaderGL) [inline www/manifest.json]: no differences

**Comment by LeaderGL** [comment]
webserver module updated with code review suggestions

**Comment by rnevet** [comment]
@laxdog @Evanito any objections to merge?

**Comment by laxdog** [comment]
Looks good.

**Pull request merged**